modules/ensemble/lib/widget/confirmation_input.dart

```diff
@@ -36,7 +36,10 @@ class ConfirmationInput extends StatefulWidget
       'fieldType': (input) =>
           _controller.fieldType = Utils.optionalString(input),
       'inputType': (type) => _controller.inputType = Utils.optionalString(type),
-      'obscureText': (type) => _controller.obscureText = Utils.optionalBool(type),
+      'obscureText': (type) {
```

Comment:
this is not needed, sorry for the backward compatibility comment. 

you can just revert to 

'obscureText': (type) => _controller.obscureText = Utils.optionalBool(type),

thanks and sorry again, approving with this change request

Reply:
Thanks khurram for making me think in that prespective as well, will surely keep that in mind if somewhere it is needed.